### PR TITLE
Adding US 163

### PR DESCRIPTION
**Pull request by YevstakhiyAndrewSyvyk**

### CODE REVIEW

File: src/test/java/com/navfort/runners/CukesRunner.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
         features = "src/test/resources/features",
         glue = "com/navfort/step_definitions",
         dryRun = false,
-        tags = "",
+        tags = "@US163",
```
Comment (mengliiii) [inline src/test/java/com/navfort/runners/CukesRunner.java:18]: Hello Andrew, one suggestion, is it possible to make the tags empty by default once everyone done your own execution? easy for other people

**Review by mengliiii — APPROVED** [review]
wow, nice work, what you gonna do next week then :) 🥇

File: src/test/java/com/navfort/runners/CukesRunner.java (hunk 1 above)
Comment (YevstakhiyAndrewSyvyk) [reply src/test/java/com/navfort/runners/CukesRunner.java:18]: You are absolutely right, I'm sorry that I missed that

File: src/test/java/com/navfort/step_definitions/US_163_StepDefinitions.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        BrowserUtils.waitForClickablility(quickLaunchpadPage.FleetDropdown, 5);
+        Actions action = new Actions(Driver.getDriver());
+        action.moveToElement(quickLaunchpadPage.FleetDropdown).perform();
+        quickLaunchpadPage.VehiclesPageLink.click();
+    }
+
+    @When("user sees all the checkboxes in the table AS")
+    public void user_sees_all_the_checkboxes_in_the_table() {
+        BrowserUtils.waitForPageToLoad(5);
+        List<WebElement> checkboxesAS = Driver.getDriver().findElements(By.xpath("//*[@data-role='select-row-cell']"));
```
Comment (oltekuteva80) [inline src/test/java/com/navfort/step_definitions/US_163_StepDefinitions.java:36]: Andrew, looks good. But I noticed that you located some of your WE inside of StepDefs file. I think it's better  to do in your VehiclesPage to follow POM design pattern

File: src/test/java/com/navfort/pages/VehiclesPage.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.navfort.pages;
+
+import com.navfort.utilities.Driver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class VehiclesPage extends QuickLaunchpadPage {
```
Comment (oltekuteva80) [inline src/test/java/com/navfort/pages/VehiclesPage.java:8]: Add your initials to the name of the class VehiclesPage_AS (on of David's prerequisites)

**Review by oltekuteva80 — COMMENTED** [review]
Andrew, good job!!!

**Pull request merged**